galaxyproject/galaxy
# ToolSourceDisplay dynamic import for shared libs bundle size improvement.

Prism is our largest individual library at ~1.5Mb parsed, but it's only used in one place.  This shifts that component to a dynamic load for its own bundle, trimming the main shared library bundle significantly (~30%) and loading it on demand.

## How to test the changes?
(Select all options that apply)
  1. View tool source, ensure that still works.
  2. Note the extra bundle fetch for ToolSourceDisplay.bundled.js and the accompanying library (something like `751.bundled.js`).
 

## License
